3p/integration.js

```diff
@@ -56,6 +56,7 @@ import {criteo} from '../ads/criteo';
 import {dotandads} from '../ads/dotandads';
 import {doubleclick} from '../ads/google/doubleclick';
 import {eplanning} from '../ads/eplanning';
+import {fakead3p} from '../ads/fakead3p';
```

Comment:
I don't have a good way to check if `getMode().localDev = true` here, because the code lies inside the iframe. I can pass the parent window mode value in or register the `fakead3p` in `amp-ad-3p-impl`. 
Or can we stripped the code out from production?